tests/tests.py

```diff
@@ -890,3 +893,98 @@ def test_invalidate_model_with_geometry(self):
         geom.save()
         # Raises ValueError if this doesn't work
         invalidate_obj(geom)
+
+
+class MultiDBInvalidationTests(BaseTestCase):
+
+    @override_settings(CACHEOPS_PREFIX=lambda q: q.db)
+    def test_bulk_create(self):
+        DbBinded.objects.cache().count()
+        DbBinded.objects.using('slave').cache().count()
+
+        db_binded = DbBinded()
+        DbBinded.objects.using('slave').bulk_create([db_binded])
+        with self.assertNumQueries(0):
+            DbBinded.objects.cache().count()
+        with self.assertNumQueries(1, using='slave'):
+            DbBinded.objects.cache().using('slave').count()
+
+    @mock.patch('cacheops.invalidation.invalidate_dict')
+    def test_bulk_update_call_invalidate(self, mock_invalidate_dict):
```

Comment:
Unneeded test of the internals.

Reply:
I would not say they are unneeded. They test a precise point : that concerned methods call invalidators with good parameters. Of course the other will fail if these one fail but they are more 'unitary'. As you wish, I can remove them.